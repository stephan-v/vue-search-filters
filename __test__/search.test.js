import Vue from 'vue/dist/vue.js';
import ExampleComponent from '../src/components/ExampleComponent.vue';

describe('ExampleComponent', () => {
    Vue.component('example-component', ExampleComponent);

    beforeEach(() => {
        document.body.innerHTML = `
            <div id="app">
                <example-component></example-component>
            </div>
        `;
    });

    it('can mount', async () => {
        await createVm();

        expect(document.body.innerHTML).toMatchSnapshot();
    });
});

async function createVm() {
    const vm = new Vue({
        el: '#app',
    });

    await Vue.nextTick(() => {});

    return { app: vm, component: vm.$children[0] };
}
