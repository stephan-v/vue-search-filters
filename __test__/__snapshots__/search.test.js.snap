// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent can mount 1`] = `
"
            <div id=\\"app\\"><div>
    I am an example component
</div></div>
        "
`;
